ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/claimantresponse/CCDResponseRejection.java

```diff
@@ -1,23 +1,21 @@
 package uk.gov.hmcts.cmc.ccd.domain.claimantresponse;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
-
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
 
-
 @Value
```

Comment:
please add true as by default it is false

Reply:
added